src/app/digital/public/api/DigitalPort.ts

```diff
@@ -0,0 +1,11 @@
+import {Port} from "core/public";
+
+import {DigitalComponent} from "./DigitalComponent";
+
+
+export interface DigitalPort extends Port {
+    readonly parent: DigitalComponent;
+
+    readonly isInputPort: boolean;
+    readonly isOutputPort: boolean;
```

Comment:
What's the thought behind `isXPort` vs something like `portDirection: "input" | "output"`?

Reply:
Probably could have both